website_geonames/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+{
+    "name": "Website Geonames",
+    "summary": "Website Geonames",
+    "author": "Moka Tourisme",
+    "website": "https://www.mokatourisme.fr",
+    "category": "Website",
+    "version": "15.0.1",
```

Comment:
```suggestion
    "version": "15.0.1.0.0",
```

I suggest to use the OCA convention for versions: `15.0` for the Odoo version + `major.minor.patch`